client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeCard/timeRangeCardStyles.ts

```diff
@@ -0,0 +1,23 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    timeRangeCard: {
+        width: '15vw',
+        height: '105%',
+        borderRadius: '1vw',
+    },
+    cardTitle: {
+        fontSize: '0.9vw',
```

Comment:
You have both variant prop and font size, which means one of them is unnecssary... I think you should delete the class

Reply:
I removed the variant